docs/installation_guide.md

```diff
@@ -131,6 +131,22 @@ also use it with any pip-based virtual environment (venv, conda, virtualenv, etc
 
 ---
 
+### Method 4: Docker
+
+#### Option A: From Docker Hub
+
+Not available yet. For now, build locally (see below).
+
+#### Option B: Build Docker Image Locally
+
+- Install [Docker](https://docs.docker.com/engine/install/)
+- Install an appropriate CUDA version (12.8 at time of writing)
```

Comment:
I'm successfully trained on 12.8, 12.9 and 13. Probably better not to mention it here? We have warnings in other parts of the doc saying 12.4+ is preferred.